### PR TITLE
Ray traced reflection render graph preparation and minor fix.

### PR DESCRIPTION
**Pull request by anisunity**
Fixed unwanted ghosting for smooth surfaces on ray traced reflections. When the surface was a perfect mirror, we still got some ghosting when enabling the denoiser (which should not happen).
Preparing ray traced reflections for the render graph support (performance and quality).

### Testing status
Ran the DXR tests locally and they were green.
Tested the anti-blur fix while changing very slighting the smoothness it looked fine.

### CODE REVIEW

**Comment by anisunity** [comment]
Adding the code for preparing render graph port for recursive rendering and a fix

File: com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/Raytracing/HDRaytracingReflection.cs [hunk 1]
```diff
@@ -168,149 +242,289 @@ DeferredLightingRTParameters PrepareReflectionDeferredLightingRTParameters(HDCam
...
+            rtrUpscaleResources.depthStencilBuffer = m_SharedRTManager.GetDepthStencilBuffer();
+            rtrUpscaleResources.normalBuffer = m_SharedRTManager.GetNormalBuffer();
+            rtrUpscaleResources.lightingTexture = lightingTexture;
+            rtrUpscaleResources.hitPointTexture = hitPointTexture;
+            rtrUpscaleResources.outputTexture = outputTexture;
+            rtrUpscaleResources.clearCoatMaskTexture = hdCamera.frameSettings.litShaderMode == LitShaderMode.Deferred ? m_GbufferManager.GetBuffersRTI()[2] : TextureXR.GetBlackTexture();
+            return rtrUpscaleResources;
+        }
+
+        void UpscaleRTReflections(CommandBuffer cmd, RTReflectionUpscaleParameters rtrUpscaleParameters, RTReflectionUpscaleResources rtrUpscaleResources)
```
Comment (JulienIgnace-Unity) [inline com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/Raytracing/HDRaytracingReflection.cs:309]: This one was probably intended to be static right?

File: com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/Raytracing/Shaders/Denoising/ReflectionDenoiser.compute [hunk 2]
```diff
@@ -67,6 +67,16 @@ void TemporalAccumulation(uint3 dispatchThreadId : SV_DispatchThreadID, uint2 gr
         return;
     }
 
+    // Decode the normal data
```
Comment (JulienIgnace-Unity) [inline com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/Raytracing/Shaders/Denoising/ReflectionDenoiser.compute:70]: I don't see it used anywhere in this PR. Was this intended to be part of it?

Comment (JulienIgnace-Unity) [reply com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/Raytracing/Shaders/Denoising/ReflectionDenoiser.compute:70]: nvm did not read properly

**Pull request merged**